lib/msal-browser/src/app/PublicClientApplication.ts

```diff
@@ -460,12 +461,12 @@ export class PublicClientApplication {
      * Default behaviour is to redirect the user to `window.location.href`.
      * @param logoutRequest 
      */
-    logout(account: AccountInfo, postLogoutRedirectUri?: string, authority?: string): void {
-        this.createAuthCodeClient(authority).then((authClient: AuthorizationCodeClient) => {
-            // create logout string and navigate user window to logout. Auth module will clear cache.
-            const logoutUri: string = authClient.getLogoutUri(account, postLogoutRedirectUri || this.getPostLogoutRedirectUri());
-            BrowserUtils.navigateWindow(logoutUri);
-        });
+    async logout(logoutRequest: EndSessionRequest): Promise<void> {
```

Comment:
I believe `logoutRequest` can be marked as optional?

Reply:
done.